### PR TITLE
Angularfy tabset and tab

### PR DESCRIPTION
**Pull request by jess0407**
https://rockabox.tpondemand.com/entity/10643

### CODE REVIEW

File: src/rb-tabs/index.js [hunk 1]
```diff
@@ -1,7 +1,8 @@
 define([
-    './rb-tabs-directive',
+    './tabset-directive',
+    './tab-directive',
     './rb-tabs.css'
-], function (rbTabsDirective, css) {
+], function (tabset, tab, css) {
```
Comment (nwhite89) [inline src/rb-tabs/index.js]: can you please make sure you keep to the naming convention here should be ``rbTabsetDirective``

File: src/rb-tabs/demo/demo.tpl.html [hunk 2]
```diff
@@ -11,9 +11,24 @@
         </rb-section>
         <rb-section heading="Default">
             <!-- <rb-component-display> -->
-                <rb-tabs></rb-tabs>
+            <tabset>
+                <tab heading="Tab 1">
```
Comment (nwhite89) [inline src/rb-tabs/demo/demo.tpl.html]: `rb-tab`

File: src/rb-tabs/demo/demo.tpl.html [hunk 3]
```diff
@@ -11,9 +11,24 @@
         </rb-section>
         <rb-section heading="Default">
             <!-- <rb-component-display> -->
-                <rb-tabs></rb-tabs>
+            <tabset>
```
Comment (nwhite89) [inline src/rb-tabs/demo/demo.tpl.html]: `rb-tabset`

**Comment by nwhite89** [comment]
I think in general there needs to be some more unit tests around this to ensure it does what is needed

**Comment by jess0407** [comment]
This is a replica of the existing AngularJS tab component out there, so basically it is how angular-ui people implement it. And they use the `href=''` and binding with the heading. To me, either way don't make a difference (as it turns out both ways the code works) so I have amended the code per suggestion.

**Comment by jess0407** [comment]
@nwhite89 I have also added some unit tests including tests for `addTab` and `select` function, let me know if this is ready to merge.

**Pull request merged**